### PR TITLE
Fix links in ambient usage and architecture docs

### PR DESCRIPTION
**Pull request by windsonsea**
Search and replace

```
](docs/
```
with
```
](/docs/
```

- [x] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [ ] Localization/Translation

### CODE REVIEW

File: content/en/docs/ambient/architecture/data-plane/index.md [hunk 1]
```diff
@@ -8,16 +8,16 @@ test: no
 
 In {{< gloss "ambient" >}}ambient mode{{< /gloss >}}, workloads can fall into 3 categories:
 1. **Out of Mesh**: a standard pod without any mesh features enabled. Istio and the ambient {{< gloss >}}data plane{{< /gloss >}} are not enabled.
-1. **In Mesh**: a pod that is included in the ambient {{< gloss >}}data plane{{< /gloss >}}, and has traffic intercepted at the Layer 4 level by {{< gloss >}}ztunnel{{< /gloss >}}. In this mode, L4 policies can be enforced for pod traffic. This mode can be enabled by setting the `istio.io/dataplane-mode=ambient` label. See [labels](docs/ambient/architecture#ambient-labels) for more details.
```
Comment (kfaseela) [inline content/en/docs/ambient/architecture/data-plane/index.md:11]: these are working for me though

**Review by kfaseela — APPROVED** [review]

File: content/en/docs/ambient/architecture/data-plane/index.md [hunk 2]
```diff
@@ -8,16 +8,16 @@ test: no
...
-1. **In Mesh, Waypoint enabled**: a pod that is _in mesh_ *and* has a {{< gloss "waypoint" >}}waypoint proxy{{< /gloss >}} deployed. In this mode, L7 policies can be enforced for pod traffic. This mode can be enabled by setting the `istio.io/use-waypoint` label. See [labels](docs/ambient/architecture#ambient-labels) for more details.
+1. **In Mesh**: a pod that is included in the ambient {{< gloss >}}data plane{{< /gloss >}}, and has traffic intercepted at the Layer 4 level by {{< gloss >}}ztunnel{{< /gloss >}}. In this mode, L4 policies can be enforced for pod traffic. This mode can be enabled by setting the `istio.io/dataplane-mode=ambient` label. See [labels](/docs/ambient/architecture#ambient-labels) for more details.
```
Comment (kfaseela) [inline content/en/docs/ambient/architecture/data-plane/index.md:11]: where are these links supposed to go? this seems to be not working? i do not see anything as ambient-labels, not ur fault though

**Comment by kfaseela** [comment]
remaining changes LGTM

File: content/en/docs/ambient/architecture/data-plane/index.md (hunk 2 above)
Comment (kfaseela) [inline content/en/docs/ambient/architecture/data-plane/index.md:11]:
```suggestion
1. **In Mesh**: a pod that is included in the ambient {{< gloss >}}data plane{{< /gloss >}}, and has traffic intercepted at the Layer 4 level by {{< gloss >}}ztunnel{{< /gloss >}}. In this mode, L4 policies can be enforced for pod traffic. This mode can be enabled by setting the `istio.io/dataplane-mode=ambient` label. See [labels](/docs/ambient/usage/add-workloads/#ambient-labels) for more details.
```

File: content/en/docs/ambient/architecture/data-plane/index.md [hunk 3]
```diff
@@ -8,16 +8,16 @@ test: no
...
+1. **In Mesh, Waypoint enabled**: a pod that is _in mesh_ *and* has a {{< gloss "waypoint" >}}waypoint proxy{{< /gloss >}} deployed. In this mode, L7 policies can be enforced for pod traffic. This mode can be enabled by setting the `istio.io/use-waypoint` label. See [labels](/docs/ambient/architecture#ambient-labels) for more details.
```
Comment (kfaseela) [inline content/en/docs/ambient/architecture/data-plane/index.md:12]:
```suggestion
1. **In Mesh, Waypoint enabled**: a pod that is _in mesh_ *and* has a {{< gloss "waypoint" >}}waypoint proxy{{< /gloss >}} deployed. In this mode, L7 policies can be enforced for pod traffic. This mode can be enabled by setting the `istio.io/use-waypoint` label. See [labels](/docs/ambient/usage/add-workloads/#ambient-labels) for more details.
```

**Review by kfaseela — APPROVED** [review]
Thanks!

File: content/en/docs/ambient/architecture/data-plane/index.md (hunk 3 above)
Comment (windsonsea) [reply content/en/docs/ambient/architecture/data-plane/index.md]: Thanks, fixed.

**Pull request merged**